### PR TITLE
fix: Incorrect configuration information read

### PR DESCRIPTION
**Pull request by zhangyue0503**
读取配置信息键名有问题

不过读对了也没用，think-migration 有问题，已经向那边提交了 issue [数据库迁移脚本向不同的连接库迁移问题](https://github.com/top-think/think-migration/issues/70)

### CODE REVIEW

**Comment by hsluoyz** [comment]
@leeqvip

File: database/migrations/20181113071924_create_rules_table.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ protected function init()
      */
     protected function getDbConfig(): array
     {
-        $default = config('tauthz.database.connection') ?: config('database.default');
+        $default = config('tauthz.enforcers.basic.database.connection') ?: config('database.default');
 
         $config = config("database.connections.{$default}");
```
Comment (leeqvip) [inline database/migrations/20181113071924_create_rules_table.php:36]: @zhangyue0503 First you should get the default `enforcer`, and then get the `database.connection` in it.
```php
$default = config('tauthz.default');
$connection = config("tauthz.enforcers.{$default}.database.connection") ?: config('database.default');

$config = config("database.connections.{$connection}");
```

**Review by leeqvip — CHANGES_REQUESTED** [review]

File: database/migrations/20181113071924_create_rules_table.php (hunk 1 above)
Comment (zhangyue0503) [reply database/migrations/20181113071924_create_rules_table.php:35]: 没毛病

**Review by zhangyue0503 — COMMENTED** [review]
没毛病，还是要走配置文件，我给直接写死了

**Comment by leeqvip** [comment]
> 没毛病，还是要走配置文件，我给直接写死了

@zhangyue0503 You seem to have forgotten to modify the last line of code.

**Review by leeqvip — APPROVED** [review]

**Pull request merged**